Azure-Samples/sample-mdw-serverless
# Add bronze2silver pipeline with notebook

## Purpose

*  Add new pipeline where silver data is saved on a different target folder based on a date column value.
* Simplify sample files removing nested json and adding new line after each json

## Does this introduce a breaking change?

```
[x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
Run pipeline 'Process Factories Data bronze2silver - Notebook' and check target folders
Pipeline run should succeed
The unprocessed files should be saved in silver container

* Test the code

```
'Process Factories Data bronze2silver - Notebook' pipeline run should succeed 
```

## What to Check
Verify that the following are valid
* ...

## Other Information
Steps missing:
Add creation of a Key Vault to bicep 
Update readme
